services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/model/ReadBatch.java

```diff
@@ -13,67 +13,87 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.extensions.dynamodb.mappingclient.operations;
+package software.amazon.awssdk.extensions.dynamodb.mappingclient.model;
 
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
 import software.amazon.awssdk.annotations.SdkPublicApi;
 import software.amazon.awssdk.extensions.dynamodb.mappingclient.BatchableReadOperation;
 import software.amazon.awssdk.extensions.dynamodb.mappingclient.MappedTableResource;
 import software.amazon.awssdk.extensions.dynamodb.mappingclient.TableMetadata;
+import software.amazon.awssdk.extensions.dynamodb.mappingclient.operations.GetItem;
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
 import software.amazon.awssdk.services.dynamodb.model.KeysAndAttributes;
 
+/**
+ * Encapsulates a single write transaction that can form a list of transactions that go into a
+ * {@link BatchGetItemEnhancedRequest}.
+ * Example:
+ *
+ * {@code
+ * ReadBatch.create(myTable, putItem(myItem));
+ * ReadBatch.createf(myTable, deleteItem(Key.create(stringValue("id123"))));
```

Comment:
Can we update this to builder style?

Reply:
Create needs to go away in fact, it doesn't have a place in the new order. Since we have a builder in this version not sure there's any justification to delay that.

Reply:
I thought we had said there would be both a builder and a create, but perhaps that was only for the requests?

Reply:
Since one of the main drivers for removing the create is the still-to-come atomic operation split this change is deferred.